src/javatests/com/google/cloud/bigquery/dwhassessment/extractiontool/db/AvroHelperTest.java

```diff
@@ -143,7 +145,11 @@ public void parseRowToAvroTest() throws Exception {
       GenericRecord result = parseRowToAvro(resultSet, testSchema);
       assertThat(result)
           .isEqualTo(
-              new GenericRecordBuilder(testSchema).set("ID", 0).set("NAME", "name_0").build());
+              new GenericRecordBuilder(testSchema)
+                  .set("ID", 0)
+                  .set("NAME", "name_0")
+                  .set("CHAR_COL", "  two  words")
```

Comment:
Should these have trailing spaces to test the effect of this?

Reply:
No, this is actually an expected value not the one that is set up and I expect spaces to be removed. But also I do not have to add trailing spaces in the setup because it's the JDBC driver that adds the spaces. Without my change the test fails here with the value "  two  words        ". These spaces increases a dump size and cause problem in on the result presentation level.

Reply:
Thanks for the explanation.